vision/mnist/ELM/ELM.jl

```diff
@@ -0,0 +1,91 @@
+#importing important libraries
+using Flux.Data.MNIST
+using Flux: onehotbatch, onecold
+using Statistics
+using Random
+
+function elmtrain(train_x, train_y, hidden_units)
+	"""
+    	function elmtrain
+    	input: 1) n * m train set; where m = #samples, n = #features
+      	     2) c * m labels; where m = #samples, c = #classes
+    	action: generates win and wout
+	"""
```

Comment:
This is a python-style docstring -- in Julia they need to be above the function

Reply:
Thank you! I have shifted the docstrings above the respective functions